src/game.cpp

```diff
@@ -5653,17 +5651,12 @@ void game::draw_critter(const Creature &critter, const point &center)
     if( !is_valid_in_w_terrain( mx, my ) ) {
         return;
     }
-    if( u.sees( &critter ) || &critter == &u ) {
+    if( u.sees( critter ) || &critter == &u ) {
         critter.draw( w_terrain, center.x, center.y, false );
         return;
     }
-    const bool has_ir = u.has_active_bionic( "bio_infrared" ) ||
-                        u.has_trait( "INFRARED" ) ||
-                        u.has_trait( "LIZ_IR" ) ||
-                        u.worn_with_flag( "IR_EFFECT" );
-    const bool can_see = m.pl_sees( u.posx, u.posy, critter.xpos(), critter.ypos(),
-                                    u.sight_range( DAYLIGHT_LEVEL ) );
-    if( critter.is_warm() && has_ir && can_see ) {
```

Comment:
Uh, we removed this a day or two ago.  Might wanna `git pull upstream master` and make sure your master branch is up-to-date.  (Test in a new branch first.  Most of the time it works in place of `git rebase` but when it doesn't, the PR's unmergeable.)